docs/ocis/release_notes.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "Release Notes"
+date: 2020-12-16T20:35:00+01:00
+weight: 0
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis
+geekdocFilePath: release_notes.md
+---
+
+## ownCloud Infinite Scale 1.0.0 Technology Preview - Initial Release Notes
+
+We are pleased to annouce the availability of ownCloud Infinite Scale 1.0.0 Technology Preview which is released as the first public version of the new Infinite Scale platform.
+
+### Microservice architecture
+
+ownCloud Infinite Scale is following the microservices architectual pattern. It is implemented as a set of microservices which are independent of each other. They are coupled with very well-defined APIs and communicate via HTTP. This architecture fosters a lot of benefits that we were going for with the new design for oCIS:
+
+- Independent services: Every service is independant, comparably small and brings it's own webserver, backend/APIs and frontend components.
+- Each service runs as a separate service on the system, increasing security and stability
+- Scalability:  High performance demands can be fulfilled by scaling the amount of services
```

Comment:
```suggestion
- Scalability:  High performance demands can be fulfilled by scaling the distribution of services
```

Maybe "distribution" or?

The "number" of different services remains the same - you just spread them onto multiple machines, or make multiple copies of them on multiple machines if one service needs more CPU... than a single machine?